daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md

```diff
@@ -199,12 +199,33 @@ await foreach (var items in subscribeConfigurationResponse.Source.WithCancellati
 }
 ```
 
+### Manage workflow instances (Alpha)
+
+```csharp
+var daprClient = new DaprClientBuilder().Build();
+
+string instanceId;
```

Comment:
Right now, this isn't valid C# because we're declaring variables and attempting to use them below without actually initializing them. Here are my suggestions for these:
```suggestion
string instanceId = "MyWorkflowInstance1;
```